target/linux/ipq40xx/patches-5.4/999-leds-add-leds-sony-mcu-driver.patch

```diff
@@ -0,0 +1,31 @@
+Index: linux-5.4.98/drivers/leds/Kconfig
+===================================================================
+--- linux-5.4.98.orig/drivers/leds/Kconfig
++++ linux-5.4.98/drivers/leds/Kconfig
+@@ -823,6 +823,13 @@ config LEDS_LM36274
+ 	  Say Y to enable the LM36274 LED driver for TI LMU devices.
```

Comment:
please, no 998- and 999- patches. Why would exactly you be the person who is last?

Reply:
Thank you, I've forgotten to check the [rule](https://openwrt.org/docs/guide-developer/build-system/use-patches-with-buildsystem) of the number in the prefix, I'll change to ```8xx - other drivers```.

Reply:
This driver should be sent upstream first

Reply:
These drivers (sony-mcu, leds-sony-mcu) are very device-specific and not generic, but should be sent...?

Reply:
Yes, does not matter that they are device specific.
They should be upstreamed.

Reply:
I see... I'll rename drivers to the device-specific name and try to send to upstream.